tools/accuracy_checker/accuracy_checker/annotation_converters/mrlEyes_2018_01.py

```diff
@@ -0,0 +1,76 @@
+"""
+Copyright (c) 2019 Intel Corporation
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import re
+from pathlib import Path
+
+from ..config import PathField
+from ..representation import ClassificationAnnotation
+from ..utils import get_path, read_txt
+
+from .format_converter import BaseFormatConverter, ConverterReturn
+
+
+class mrlEyes_2018_01_Converter(BaseFormatConverter):
+    """
+    mrlEyes_2018 dataset converter. 
+    """
+
+    __provider__ = 'mrlEyes_2018_01'
+    annotation_types = (ClassificationAnnotation, )
+
+    @classmethod
+    def parameters(cls):
+        configuration_parameters = super().parameters()
+        configuration_parameters.update({
+            'data_dir': PathField(is_directory=True, description="Path to mrlEyes_2018_01 dataset root directory.")
+        })
+
+        return configuration_parameters
+
+    def configure(self):
+        """
+        This method is responsible for obtaining the necessary parameters
+        for converting from the command line or config.
+        """
+        self.data_dir = self.config['data_dir']
+        self.use_every = self.config['use_every']
```

Comment:
`use_every` should be added to parameters. Also I think you need to make it numeric to specify interval between testing images and it will be useful if you make it optional.

Reply:
Katya, I think we should use fixed dataset for test. Ok, someone had asked me to add 'use every'.  Why we need specify this number? 
And one more note - this is dataset was used for training. Because we have only one approved dataset.  And I separated it to train and trainval by %10 rule.

Reply:
1. Anyway you need register this parameter, because now if somebody does not specify this, annotation conversion failed with `KeyError`
2. I understand you, you can made 10 is default to guarantee that 10th image will be used even parameter is not provided.
3. it just care about future, if something changed (e.g. we get another dataset or need  reduce test split), it will be easier to maintain if you add this flexibility

Reply:
@ygrnn123, I looked on conversation history from old PR, It was not my comment about `use_every`, so you can just remove it if you think that we do not need this option

Reply:
Yes, there was not your comment. ) Ok, I think, this option is not required.